CryptKeeperZK/crypt-keeper-extension
# feat: multi accounts support

## Explanation

This PR adds support for multi account support.

Details are below:
- [x] Add account storage to wallet service
- [x] Merge backup data for account storage
- [x] Add account field to message for signing
- [x] Add account to identity view
- [x] Add lodash dependency

## More Information

Blocked by #373 
Related to #377
Closes #383 

## Screenshots/Screencaps

![image](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/14254374/d3dc67c6-2bae-4eb9-8c47-9527805c929c)

## Manual Testing Steps

1. Reinstall extension
2. Complete onboarding
3. Create identity with metamask
4. See account address for interrep identities created with metamask 

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
